cli/config/configuration-qemu.toml.in

```diff
@@ -97,6 +97,32 @@ default_memory = @DEFMEMSZ@
 # 9pfs is used instead to pass the rootfs.
 disable_block_device_use = @DEFDISABLEBLOCK@
 
+# Shared file system type:
+#   - virtio-9p (default)
+#   - virtio-fs
```

Comment:
should we mention that a QEMU version greater than X is required to use this filesystem ?

Reply:
It is currently not available in upstream QEMU.  Launching the guest will fail with a QEMU error message because -device vhost-user-fs-pci is not supported.  This can be seen in the kata-runtime log.

It is possible to probe a QEMU binary for this feature by running "qemu-system-x86_64 -device ?" to list available device models.  I'm not sure if that's a good idea though since it will slow down Kata.  Caching this information somewhere makes it more complex and prone to issues.  I hope just relying on the QEMU error message is sufficient.